### PR TITLE
LPS-157444 [CP2.0]  Add Specific behaviour enhancement for liferay staff tests

### PR DESCRIPTION
**Pull request by luanakarla**

### CODE REVIEW

**Comment by luanakarla** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2Utils.path [hunk 1]
```diff
@@ -85,6 +85,11 @@
 	<td>//*[contains(@class, 'modal-content')]//*[text()='${key_text}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TEXT</td>
+	<td>//*[contains(text(), '${key_text}')]</td>
+	<td></td>
+</tr>
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2Utils.path:92]: Hi @luanakarla would you please be more descriptive on this path? TEXT is too wide.

Comment (luanakarla) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2Utils.path:92]: Yea, done!

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:sf

**Pull request closed without merging**